Podfile.lock

```diff
@@ -45,4 +45,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d13bb859974d75763866520e0313eec1e5f76228
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```

Comment:
Super take it or leave it comment - if we wanted to move the CocoaPods version bump out of this PR to make this PR easily revertible, in case we do add back Preferred Payment Methods.

Reply:
UGH. This wasn't supposed to happen! I'll take it out.